backend/src/packages/auth/auth.service.ts

```diff
@@ -17,9 +17,11 @@ class AuthService {
 
   private async checkExistingUser({
```

Comment:
```suggestion
  private async checkIsExistingUser({
```
Idea is `const isExistingUser = checkIsExistingUser(...`